wishlist.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sloshed</title>
+    <link rel="stylesheet" href="https://crave-ui.netlify.app/styles/main.css" />
+    <script src="https://kit.fontawesome.com/742ff825de.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./styles/main.css">
+</head>
+<body>
+    <div class="wrapper-products">
+        <div class="header">
+            <div class="brand">
+                <a href="/index.html"><h2>Sloshed.</h2></a>
+                <p>Get your booze before you snooze</p>
+                <p>10 mins delivery of your favourite alcohol</p>
+            </div>
+            <input type="text" class="search-bar" cols="20" rows="2" placeholder="🔍Search"></input>
+            <div class="navbar">
+            <a href="login.html"><button class="btn-login">Login</button></a>
+                <div class="badge-container wishlist">
+                    <a href="/wishlist.html" class="fa fa-regular fa-heart"></a>
+                    <span class="badge-icon-img  icon">8</span>
+                </div>
+                <div class="badge-container cart">
+                    <a href="/cart.html" class="fa fa-cart-shopping"></a>
+                    <span class="badge-icon-img  icon">2</span>
+                </div>
+            </div>
+        </div>
+        <div class="products-container">
+            <div class="products-listings">
+                <h1 class="products-title">Wishlist</h1>
+                <div class="card card-image">
+                    <img class="card-img" src="./assets/bira-card.jpg" alt="Card image">
```

Comment:
for image use / before closing the tag

Reply:
Are you sure this is a good practice? I checked MDN as well: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img

Reply:
As you head towards React you will get problems with this type of convention